doc/modules/ROOT/pages/index.adoc

```diff
@@ -19,11 +19,11 @@ This example focuses on the security features and assumes that you have some gen
 
 * Hazelcast on Kubernetes:
 ** https://github.com/hazelcast/hazelcast-kubernetes[Hazelcast Kubernetes README]
-** https://github.com/hazelcast/hazelcast-code-samples/tree/master/hazelcast-integration/kubernetes[Hazelcast Kubernetes Code Sample]
-** https://github.com/hazelcast/hazelcast-code-samples/tree/master/hazelcast-integration/kubernetes/samples/embedded[Hazelcast Kubernetes Embedded Code Sample]
+** https://guides.hazelcast.org/kubernetes/[Hazelcast Kubernetes Guide]
+** https://guides.hazelcast.org/kubernetes-embedded/[Hazelcast Kubernetes Embedded Guide]
```

Comment:
I'd prefer to use relative links to guides if we somehow decide to change their urls at some point.
```suggestion
** xref:kubernetes:ROOT:index.adoc[Hazelcast on Kubernetes Guide]
** xref:kubernetes-embedded:ROOT:index.adoc[Embedded Hazelcast on Kubernetes Guide]
```

Reply:
Good point. Will change the same for `kubernetes-wan` guide